influent-java/src/main/java/influent/forward/ForwardClient.java

```diff
@@ -0,0 +1,80 @@
+package influent.forward;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Client ip/network authentication & per_host shared key
+ */
+public class ForwardClient {
+  private String host = null;
+  private String network = null;
+  private String sharedKey = null;
+  private List<String> usernames = new ArrayList<>();
+
+  public ForwardClient(String host,
+                       String network,
+                       String sharedKey,
+                       List<String> usernames) {
+    this.host = host;
+    this.network = network;
+    this.sharedKey = sharedKey;
+    this.usernames = usernames;
+  }
+
+  public static class Builder {
+    private String host;
+    private String network;
+    private String sharedKey;
+    private List<String> usernames;
+
+    /**
+     * Set the IP address or host name of the client
+     *
+     * @param host The IP address or host name of the client
+     * @return this builder
+     */
+    public Builder host(String host) {
+      this.host = host;
+      return this;
+    }
+
+    /**
+     * Set network address specification
+     *
+     * @param network Network address specification
+     * @return this builder
+     */
+    public Builder network(String network) {
```

Comment:
What is difference between host and network?

Reply:
https://github.com/fluent/fluentd/blob/v0.14.20/lib/fluent/plugin/in_forward.rb#L118-L142

Fluentd's in_forward treat that `host` and `network` are exclusive each other.
`host` means that client source IP address (or calculated from hostname). ex. `example.com` or `192.168.1.11`.
`network` means that client source IP address range. ex. `192.168.1.0/32`

For example, If `network = "192.168.1.0/24"`:
* client's IP address is `192.168.1.1`: OK
* client's IP address is `192.168.2.1`: NG (refused by server)